src/components/DataDecorator/DataDecorator.js

```diff
@@ -55,7 +55,21 @@ class DataDecorator extends Component {
       title,
       scoreThresholds,
       scoreDescription,
+      onClick: propOnClick,
+      onContextMenu: propOnContextMenu,
     } = this.props;
+    const onClick = !propOnClick
+      ? event => {
+          event.stopPropagation();
+          this.toggleOpen();
+        }
+      : propOnClick;
```

Comment:
For the stopPropagation I'm not 100% why it was needed. In my testing everything looks fine if I include a basic logging function for `onClick` with propagation _not_ stopped. I'd have to do a rebuild of the addons with it removed from the default implementation of `onClick` to see if it has something to do with toggling the panel.

So, I originally changed this because I thought there was a use case for a consumer to override the default behavior completely, i.e., opening the panel! But as I mentioned, this is no longer needed so the change could completely be removed.

Reply:
I removed the option to override this default.